frontend/src/Student/components/ShowSubmissionForm.js

```diff
@@ -0,0 +1,467 @@
+import React, { Component, Fragment } from 'react'
```

Comment:
According to @JohnPMurray's commit message, this is an abstract form just not in use yet.

Reply:
Based off your comments above, about rethinking the refactor, I'm going to just delete it for now.